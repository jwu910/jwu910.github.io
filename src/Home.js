import React from 'react';

import NavBar from './js/components/navigation/NavBar';
// import nav bar
// import profile picture
// import about snippet

const Home = (props) => (
  <div>
    <NavBar />
    <h1>Hello there</h1>
  </div>
);

export default Home;