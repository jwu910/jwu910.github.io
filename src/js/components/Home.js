import React from 'react';

import { Link } from 'react-router-dom';
// import nav bar
// import profile picture
// import about snippet

class Home extends React.Component {
  render() {
    const currentPath = window.location.search.split('=')[1];

    if (currentPath) {
      window.location.path = currentPath;
    }

    return (
      <div className="home-container">
        <h3>We're technically still under construction...</h3>
        <h4>
          But please check out some of my
          <a href="/portfolio">current projects!</a>
        </h4>
      </div>
    );
  }
}

export default Home;
