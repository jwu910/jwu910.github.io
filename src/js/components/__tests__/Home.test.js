import React from 'react';
import renderer from 'react-test-renderer';

import Home from '../Home';

describe('Snapshots', () => {
  it('Renders correctly', () => {
    const component = renderer
      .create(
          <Home />
      )
      .toJSON();

    expect(component).toMatchSnapshot();
  });
});