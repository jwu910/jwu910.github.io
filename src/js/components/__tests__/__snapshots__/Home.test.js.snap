// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshots Renders correctly 1`] = `
<div
  className="home-container"
>
  <h3>
    We're technically still under construction...
  </h3>
  <h4>
    But please check out some of my
    <a
      href="/portfolio"
    >
      current projects!
    </a>
  </h4>
</div>
`;
